lib/content-services/upload/components/base-upload/upload-base.ts

```diff
@@ -57,21 +59,43 @@ export abstract class UploadBase {
     @Input()
     nodeType: string = 'cm:content';
 
+    /** @deprecated 2.4.0 Use UploadService.fileUploadComplete instead */
     /** Emitted when the file is uploaded successfully. */
     @Output()
     success = new EventEmitter();
 
-    /** @deprecated 2.4.0 */
+    /** @deprecated 2.4.0 No longer used by the framework */
```

Comment:
Why deprecate success and error event?

Reply:
We have a bunch of dedicated events in the Upload Service, I think it's easier to focus on those.

```
queueChanged: Subject<FileModel[]> = new Subject<FileModel[]>();
    fileUpload: Subject<FileUploadEvent> = new Subject<FileUploadEvent>();
    fileUploadStarting: Subject<FileUploadEvent> = new Subject<FileUploadEvent>();
    fileUploadCancelled: Subject<FileUploadEvent> = new Subject<FileUploadEvent>();
    fileUploadProgress: Subject<FileUploadEvent> = new Subject<FileUploadEvent>();
    fileUploadAborted: Subject<FileUploadEvent> = new Subject<FileUploadEvent>();
    fileUploadError: Subject<FileUploadErrorEvent> = new Subject<FileUploadErrorEvent>();
    fileUploadComplete: Subject<FileUploadCompleteEvent> = new Subject<FileUploadCompleteEvent>();
    fileUploadDeleted: Subject<FileUploadDeleteEvent> = new Subject<FileUploadDeleteEvent>();
    fileDeleted: Subject<string> = new Subject<string>();
```

Reply:
removed deprecation for both events